handbook/company/pricing-features-table.yml

```diff
@@ -283,7 +283,7 @@
 #  ╚═╝╚═╝╚═╝╩╚═  ╩ ╩╚═╝╚═╝╚═╝╚═╝╝╚╝ ╩   ╚═╝ ╩ ╝╚╝╚═╝
 - industryName: User account sync
   description: Sync user accounts via Okta, AD, or any IDP. 
-  documentationUrl: https://fleetdm.com/docs/using-fleet/mdm-macos-setup-experience
+  documentationUrl: https://fleetdm.com/docs/deploy/single-sign-on-sso#single-sign-on-sso
```

Comment:
```suggestion
  documentationUrl: https://fleetdm.com/guides/macos-setup-experience#end-user-authentication-and-end-user-license-agreement-eula
```

I think this line in the pricing page is about auto populating local user accounts. This is the section we want to link to:

![Screenshot 2025-01-20 at 10 01 45 AM](https://github.com/user-attachments/assets/9954392a-7191-4223-8897-01373ddd05cf)

Currently, it's only macOS so I updated the description.